include/crypto/md32_common.h

```diff
@@ -63,6 +63,10 @@
  *      #define HASH_BLOCK_DATA_ORDER   md5_block_data_order
  */
 
+#ifndef OSSL_CRYPTO_MD32_COMMON_H
+# define OSSL_CRYPTO_MD32_COMMON_H
+# pragma once
+
 #include <openssl/crypto.h>
```

Comment:
Style NIT: we indent preprocessor directives, so all preprocessor lines inside the new #ifndef should gain a space.

Reply:
That's true in principle, but there are other locations where include guards were added afterwards and not the entire indentation of the preprocessor directives was adjusted. That's why I didn't bother to request it. (I checked the fact, because I had the same idea).

Reply:
But sure, if @liwg06 would volunteer to fix it, that would be great!

Reply:
I clone the repo again to dismiss the intermediate results of local compilation, then search the current practice for define guard:
`grep -rn ./openssl/ --include=*.h -e "#ifndef.*_H$" -A 1|grep "# define "|wc -l`
271
`grep -rn ./openssl/ --include=*.h -e "#ifndef.*_H$" -A 1|grep "#define "|wc -l`
8
It shows that (may be not accurate) 271 define guards are followed by "# define xxx", and 8 by "#define xxx", so this patch consistent with the majority.

I'm not sure whether I got what you talk about. @paulidale  @mspncp 
Is it change pattern A to pattern B?
**_patern A_**:
```
#ifndef OSSL_XXX_H
# define OSSL_XXX_H
# pragma once
```
**_patern B_**:
```
#ifndef OSSL_XXX_H
#define OSSL_XXX_H
#pragma once 
```
If so, I'll update the commit.

Reply:
But as you already noticed, there are places where the indentation is not as it should be.

Reply:
@liwg06 What we are asking for is to indent the preprocessor directives further in the source code if you are adding another #ifndef around it.

Reply:
Got it. 
I use the script to indent the line begien with "#" in target files at first, and then add the define guard.
```
for filename in $(ls \
 include/crypto/md32_common.h \
 include/internal/der.h \
 include/internal/tsan_assist.h \
 providers/implementations/include/prov/ciphercommon.h \
 providers/implementations/include/prov/ciphercommon_aead.h \
 providers/implementations/include/prov/ciphercommon_ccm.h \
 providers/implementations/include/prov/ciphercommon_gcm.h ) ; \
do cat $filename | sed 's/^#/# /' > $filename.new.h ; mv $filename.new.h $filename ; done
```
The commit is updated, I will check the CI result later.